servicetalk-loadbalancer/src/test/java/io/servicetalk/loadbalancer/RandomSubsetterTest.java

```diff
@@ -71,7 +72,7 @@ private static Set<PrioritizedHost> subsetSet(int randomSubsetSize, List<Priorit
     }
 
     private static List<PrioritizedHost> subset(int randomSubsetSize, List<PrioritizedHost> hosts) {
-        return new RandomSubsetter(randomSubsetSize).subset(hosts);
+        return new RandomSubsetter.RandomSubsetterFactory(randomSubsetSize).build("").subset(hosts);
```

Comment:
Is it a factory or a builder? looks a bit confusing to see a build method on a factory.
Consider either renaming to builder or use "create"/"newSubsetter"/etc. method

Reply:
Good point. Let's go with `newSubsetter`.